build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -703,4 +711,4 @@ export function useUnloading(handler: (state: State, data: any, url: string, his
  * Registers callback for when this Scene has navigated to another Scene
  * @param handler The unloaded event handler
  */
-export function useUnloaded(handler: (state: State, data: any, stateContext: StateContext) => void) : void;
+export function useUnloaded(handler: (state: State, data: any, stateContext: StateContext) => void) : void;
```

Comment:
Can you remove this change so this file doesn't show up in the diff, please? Looks like you've added a new line